### ISSUE TITLE
'break' command doesn't work on methods defined inside pry

### ISSUE THREAD

**Issue by banister** [nixme/pry-debugger#14]
``` ruby
[11] (pry) main: 0> def little_pig
[11] (pry) main: 0*   1
[11] (pry) main: 0*   2
[11] (pry) main: 0*   3
[11] (pry) main: 0* end  
=> nil
[12] (pry) main: 0> break little_pig
ArgumentError: Invalid file!
from /Users/john/.rvm/gems/ruby-1.9.3-p194/gems/pry-debugger-0.2.0/lib/pry-debugger/breakpoints.rb:13:in `add'
[13] (pry) main: 0> 
```

**Comment by rking** [comment]
+1

**Comment by banister** [comment]
Note I have a local fix for this

**Comment by nixme** [comment]
Fixed in #21
